### PR TITLE
MM-20566 Migrate 'components/admin_console/system_users/list' to TypeScript.

### PR DESCRIPTION
**Pull request by anchepiece**
#### Summary
This pull request migrates the 'components/admin_console/system_users/list' module to TypeScript.
Tests pass locally, suggestions and improvements welcome.

#### Ticket Link
Fixes mattermost/mattermost-server#13506
JIRA https://mattermost.atlassian.net/browse/MM-20566

### CODE REVIEW

File: components/admin_console/system_users/list/system_users_list.tsx [hunk 1]
```diff
@@ -1,10 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
+// import PropTypes, { string } from 'prop-types';
```
Comment (ali-farooq0) [inline components/admin_console/system_users/list/system_users_list.tsx:4]: should be removed

**Review by ali-farooq0 — APPROVED** [review]
Thanks for your contribution @anchepiece 🎉

File: components/admin_console/system_users/list/system_users_list.tsx (hunk 1 above)
Comment (anchepiece) [reply components/admin_console/system_users/list/system_users_list.tsx]: Thanks!

**Review by nevyangelova — APPROVED** [review]
LGTM!

**Review by stevemudie — APPROVED** [review]
LGTM

**Comment by hanzei** [comment]
Another one! :rocket:

**Pull request merged**